include/z64.h

```diff
@@ -909,7 +909,7 @@ typedef struct GlobalContext {
     /* 0x11E04 */ s16* setupExitList;
     /* 0x11E08 */ Path* setupPathList;
     /* 0x11E0C */ ElfMessage* cUpElfMsgs;
-    /* 0x11E10 */ char unk_11E10[0x4];
+    /* 0x11E10 */ void* specialEffects;
```

Comment:
"special" is an alright name, though I do think that `customEffects` moreso gets the point across that you can use this if none of the effects supplied to you by the engine are sufficient. It lets you make your own if the provided ones arent what you need

Reply:
curious what others think tho

Reply:
"special" seems fine, tho it might not be the best. And I don't think it really matters anyway, since at the end of the day it was just used as a generic pointer to hold various things.